pandas/io/parsers/readers.py

```diff
@@ -446,10 +446,10 @@
     "low_memory",
 }
 
-_deprecated_defaults: dict[str, Any] = {
-    "error_bad_lines": None,
-    "warn_bad_lines": None,
-    "squeeze": None,
+_deprecated_defaults: dict[str, tuple[Any, str]] = {
+    "error_bad_lines": (None, "Use on_bad_lines in the future."),
```

Comment:
Could the `None`s just be replaced with the error message and the downstream code refactored appropriately?

Reply:
The None initializes the default value of the argument. We have to kepe it, if we want to keep the structure generic for the future.

Reply:
Gotcha. A comment describing a format, or use of a Namedtuple, would be great to clarify that here.

Reply:
Added a namedtuple, makes the access easiert too